engine/common/model.c

```diff
@@ -303,7 +303,7 @@ model_t *Mod_LoadModel( model_t *mod, qboolean crash )
 		loaded = true;
 		break;
 	case Q1BSP_VERSION:
-	case HLBSP_VERSION:
+	case LittleLong(HLBSP_VERSION):
```

Comment:
I suggest you to change this:
```
uint32_t version;

version = LittleLong( *(uint *)buf );
```

So you don't have to add macro calls to every version check.